addons/account_edi_ubl_cii/models/account_edi_format.py

```diff
@@ -123,14 +123,14 @@ def _ubl_cii_post_invoice(self, invoice):
         if self.code not in ['facturx_1_0_05', 'efff_1']:
             attachment_create_vals.update({'res_id': invoice.id, 'res_model': 'account.move'})
 
-        attachment = self.env['ir.attachment'].create(attachment_create_vals)
+        attachment = self.env['ir.attachment'].with_user(SUPERUSER_ID).create(attachment_create_vals)
 
         res = {invoice: {'attachment': attachment}}
         if errors and self.code == 'facturx_1_0_05':
             res[invoice].update({
                 'success': False,
-                'error': _("Errors occured while creating the EDI document (format: %s). The receiver "
-                           "might refuse it.", builder._description)
+                'error': _("Errors occured while creating the EDI document (format: %s)."
+                           "The receiver might refuse it.", builder._description)
```

Comment:
This is breaking the translations because of one space difference

Reply:
Aie aie aie, sorry, change is done!

Thanks & Bisous!